cryosmsSup/cryosms.db

```diff
@@ -390,6 +411,75 @@ record(bo, "$(P)START:SP")
     field(SDIS, "$(P)DISABLE")
 }
 
+### SWITCH STATUS in order of processing
+
+record(ai, "$(P)SWITCH:TEMP")
+{#first, read from the sim record/temp val on thermostat
+    field(DESC, "Temperature of switch")
+    field(INP, "$(SWITCH_TEMP_PV=$(P)SIM:SWITCH:TEMP)")
+    field(FLNK, "$(P)SWITCH:STAT:NOW:OLD.PROC")
+    field(SCAN, "1 second")
+}
+
+record(mbbi, "$(P)SWITCH:STAT:NOW:OLD")
+{#store the old value of stat:now
+    field(SCAN, "Passive")
+    field(INP, "$(P)SWITCH:STAT:NOW")
+    field(ZRST, "Cold")     field(ZRVL, "0")
+    field(ONST, "Changing") field(ONVL, "1")
+    field(TWST, "Warm")     field(TWVL, "2")
+    field(FLNK, "$(P)SWITCH:STAT:NOW:CALC PP MS")
+}
```

Comment:
PP and MS have no meaning on FLNK, they only apply to OUT

Reply:
If you want to pass severity, you will need to use FLNK to trigger the next record which then reads the severity via its input link field (specifying MS)